meta-resin-common/recipes-core/packagegroups/packagegroup-resin-connectivity.bb

```diff
@@ -6,7 +6,7 @@ PR = "r0"
 inherit packagegroup
 
 # By default resin uses connman as network manager.
-NETWORK_MANAGER_PACKAGES ?= "connman connman-client"
+NETWORK_MANAGER_PACKAGES ?= "connman connman-client networkmanager modemmanager"
```

Comment:
@michal-mazurek installing `networkmanager` and `modemmanager` like this doesn't compile the right support in NetworkManager to integrate with ModemManager. We only need to specify `networkmanager` here and enable specific PKGCONFIGs that will automatically pull in `modemmanager`. The available PKGCONFIGS of `networkmanager` can be found in the recipe[1].

It looks like we need `concheck`, `modemmanager`, `ppp`, and `dnsmasq`.

Also, we need to enable two PKGCONFIGs for `modemmanager`[2]. Namely `mbim` and `qmi`.

[1] http://cgit.openembedded.org/meta-openembedded/tree/meta-oe/recipes-connectivity/networkmanager/networkmanager_1.0.12.bb#n48
[2] http://cgit.openembedded.org/cgit.cgi/meta-openembedded/tree/meta-oe/recipes-connectivity/modemmanager/modemmanager_1.6.0.bb?h=master